content/release-10-16-0/devices-10-16-0-bundle/10-16-0-0.md

```diff
@@ -248,4 +269,10 @@ OPCUA</td>
 DM-822</td>
 </tr>
 
+<tr>
+<td>OPC UA</td>
+<td>Due to a change introduced in 10.14, the opcua-device-gateway on OPC UA server connects or reconnects did a full device type matching with a comparison of device types to the server's address space to find matched nodes and the application of these matched nodes. With this fix, the opcua-device-gateway directly applies the matched nodes settings if there was a matching done previously and no more recent updates to the device types were done.</td>
```

Comment:
"the opcua-device-gateway on OPC UA server" --> this misses an article for me. What do you think @BeateRixen?

Reply:
It says "OPC UA server connects or reconnects". This is one term, connects/reconnects are not verbs. It is a bit hard to read but correct I would say.

Reply:
Ok, thanks for clarifying.